src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentById(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId)
+    || null;
```

Comment:
null is redundant here, filter always returns the array

Reply:
.Okay, but what is the word "nit"?

Reply:
<img width="609" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/70220667/2aa891bb-d4e5-414e-955b-f0f13f9f0b1a">